src/common.rs

```diff
@@ -90,6 +93,8 @@ pub trait CalcConsensus<'a, R: SequenceRead> {
                 .unwrap())
                 .join(",")
         )
+        .as_bytes()
+        .to_vec()
```

Comment:
This format!-Macro does not really make sense as the resulting string is transformed to a byte vector afterwards.
I tried to directly work on byte-arrays but without success.
Any idea on how to refactor this as I do not want this to be merged.